src/App.tsx

```diff
@@ -1,14 +1,75 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos, getUser } from './api';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import { Status } from './types/Statuses';
+
+const applyFilter = (todos: Todo[], filter: Status) => {
+  const conditions = {
+    [Status.All]: () => {
+      return todos;
+    },
+    [Status.Active]: () => {
+      return todos.filter(todo => !todo.completed);
+    },
+    [Status.Completed]: () => {
+      return todos.filter(todo => todo.completed);
+    },
+  };
+
+  return conditions[filter]();
+};
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [user, setUser] = useState<User | null>(null);
+  const [loadingTodos, setLoadingTodos] = useState(true);
```

Comment:
Rename all boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isLoadingTodos, setIsLoadingTodos] = useState(true);
```